vnext/Microsoft.ReactNative/Modules/AppStateModule.cpp

```diff
@@ -52,7 +76,7 @@ void AppState::GetCurrentAppState(
     std::function<void(AppStateChangeArgs const &)> const &success,
     std::function<void(React::JSValue const &)> const &error) noexcept {
   AppStateChangeArgs args;
-  args.app_state = m_active ? "active" : "background";
+  args.app_state = m_enteredBackground ? "background" : (m_deactivated ? "inactive" : "active");
```

Comment:
nit, could factor out this line:
```cpp
m_enteredBackground ? "background" : (m_deactivated ? "inactive" : "active")
```

into a helper, since you're repeating it in 3 places

Reply:
Solid! I've factored it out to GetAppState private method :100: